Game Vending Machine & Paper For Civvies

nearlyNonexistent opened the pull request:
Added a vending machine to the lounge.

![capture](https://user-images.githubusercontent.com/4041565/28890862-26855bd2-778e-11e7-9d7d-e3e06c75d6f1.PNG)



It dispenses gaming materials such as dice, gaming dice, cards, chess and checker pieces, and some of the boring prizes from the arcade machines (not the good ones). With the contraband wire you can get snap pops or the clown's water flower, and with a coin you can get a cap gun and ammo for it.

I also added a paper bin to the lounge and one to the mess hall for civilian usage.

Fixes #17479

Techhead0 commented:
Dunno if the mess hall is the best place for it. Might be better in or around the lounge area.

nearlyNonexistent commented:
I put it in the mess hall mostly because it had a stool that was easy to replace and no one ever really used, to be honest.

nearlyNonexistent commented:
I'm gonna change out the sprite and possibly move it into the lounge if someone can find a good corner to fit it in, the lounge is a lot smaller.

nearlyNonexistent commented:
Moved it to the lounge and changed the sprite to one made by ChrimsonShrike on discord. Editing first post with new details and in-date picture.

nearlyNonexistent commented:
Word of Mod says return it to sacrificing the mess hall stool. IT SHALL BE MOVED BACK.

nearlyNonexistent committed:
Adds paper bins to the mess hall and lounge, and new vending machine

Adds a working vending machine that sells game materials for the crew, in the mess hall. Sacrificed a stool for this. Adds paper bins to the mess hall and lounge.

nearlyNonexistent commented:
Map is cleaned, code is ready, sprite is good, everything compiles. I'll add a changelog and this can be merged. Officially ready for merge as soon as someone approves it.
